post.md

```diff
@@ -0,0 +1,346 @@
+---
+post_title: Flashing MinnowBoard Turbot with Raspberry Pi Zero W
+author: Piotr Król, Arkadiusz Cichocki
+post_excerpt: ""
+layout: post
+published: true
+post_date: 2017-11-20 00:21:00
+tags:
+  - coreboot
+  - UEFI
+  - RPi
+  - Intel
+  - BayTrail
+categories:
+  - Firmware
+---
+
+Recently we started preparation of coreboot training for one of our customers.
+Our platform of choice for that training is MinnowBoard Turbot. There are
+couple reasons for that:
+
+* during training we can show recent firmware trends - despite we don't like
```

Comment:
For reasons of formatting I would treat every point as a sentence. So I would change `during` to `During` here.